app/models/user.js

```diff
@@ -151,9 +152,5 @@ export default Model.extend({
     user = JSON.parse(this.get('sessionStorage').getItem('travis.user'));
     user[name.underscore()] = this.get(name);
     return this.get('sessionStorage').setItem('travis.user', JSON.stringify(user));
-  },
-
-  avatarUrl: Ember.computed('email', function () {
-    return this.get('externalLinks').gravatarImage(this.get('email'), 36);
```

Comment:
I think that with this removal we could also remove `externalLinks: service()` from the beginning of the class.